dff/script/core/message.py

```diff
@@ -155,37 +161,37 @@ def validate_source(cls, value):
 class Audio(DataAttachment):
     """Represents an audio file attachment."""
 
-    pass
+    type: Literal["audio"] = "audio"
 
 
 class Video(DataAttachment):
     """Represents a video file attachment."""
 
-    pass
+    type: Literal["video"] = "video"
 
 
 class Animation(DataAttachment):
     """Represents an animation file attachment."""
 
-    pass
+    type: Literal["animation"] = "animation"
 
 
 class Image(DataAttachment):
     """Represents an image file attachment."""
 
-    pass
+    type: Literal["image"] = "image"
 
 
 class Sticker(DataAttachment):
     """Represents a sticker as a file attachment."""
 
-    pass
+    type: Literal["sticker"] = "sticker"
 
 
 class Document(DataAttachment):
     """Represents a document file attachment."""
 
-    pass
+    type: Literal["document"] = "document"
 
 
 class Message(DataModel):
```

Comment:
Also, `Message.attachments` annotation should be updated to `List[Union[Poll, Image, ...]]`.
I'm pretty sure serialization won't work properly without it.